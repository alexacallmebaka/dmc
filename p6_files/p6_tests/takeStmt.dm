funct : () void {
  x : perfect int;
  give x;
  take x;
  isCold : bool = too hot;
}