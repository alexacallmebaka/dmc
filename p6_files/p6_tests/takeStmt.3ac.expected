[BEGIN GLOBALS]
funct
[END GLOBALS]
[BEGIN funct LOCALS]
x (local var of 8 bytes)
isCold (local var of 8 bytes)
[END funct LOCALS]
fun_funct:  enter funct
WRITE [x]
READ [x]
[isCold] := 0
lbl_0:      leave funct
